### ISSUE TITLE
Factor: Womens travel patterns - Subfactor: Location of primary schools 

### ISSUE THREAD

**Issue by timlinux** [kartoza/GEEST#40]
````
[out:xml] [timeout:25];
 {{geocodeArea:Saint Lucia}} -> .area_0;
(
    node["amenity"="school"](area.area_0);
    way["amenity"="school"](area.area_0);
    relation["amenity"="school"](area.area_0);
);
(._;>;);
out body;
````
This factor is assessed through network analyses to calculate catchment areas using Open Route Service. After each analysis, the territory is divided into 100m x 100m rasters, and the factor score is calculated as the average score within each raster. The scoring for each type of facility is categorized by distance as follows:

0 to 400 meters: 5
401 to 800 meters: 4
801 to 1200 meters: 3
1201 to 1500 meters: 2
1501 to 2000 meters: 1
Over 2000 meters: 0
Source from OSM, Humdata

**Comment by carolinamayh** [comment]
NOTE: The original methodology for assessing Women's Travel Patterns is incorrect. The score for this factor should be an average of the accessibility scores for each subfactor (green spaces, grocery stores, pharmacies, kindergartens, and primary schools). As such, each point type should be included separately to allow for individual subfactor calculations. It is not the presence of any single factor nearby that determines the score for the rasters but rather the average score from all nearby features, including those that are absent. Therefore, the denominator should remain constant (4). @osundwajeff
Primary schools and kindergartens should be considered as a single feature.

**Comment by dragosgontariu** [comment]
@osundwajeff @ClaraIV @carolinamayh @mvmaltitz

Hi Jeff,

Could you please rename the output "Facility_Kindergarten_Primary_Schools?"

Currently the output is named generic Facility_X and this is confusing when plotted against the input data.



![Image](https://github.com/user-attachments/assets/851b3aa2-3d71-45b0-98b6-df07265ee94d)

Thanks,
Dragos

**Comment by ClaraIV** [comment]
@osundwajeff I linked this to ticket #75, please see specifications there. I mentioned they are changed, once we reviewed it all. 
Please follow the specifications there for the renaming, sorry for any confusion.
